### PR TITLE
Facebook link added in contribution guide section

### PR DESCRIPTION
**Pull request by Nargis21**
Issue No: #65
Path: Contribution Guide -> Areas of Contribution
On that page, I added "Facebook" also as a social media contribution section along with Twitter and LinkedIn.

### CODE REVIEW

File: docs/How-To-Guides/how-to-create-play.md [hunk 1]
```diff
@@ -90,6 +92,8 @@ npx create-react-play@latest -c <the_play_id>
   npm run start
   ```
 
+```
```
Comment (supminn) [inline docs/How-To-Guides/how-to-create-play.md:95]: These backticks on lines 95 and 126 need to be removed. It's not getting rendered as expected.
Preview - https://deploy-preview-66--react-play-docs.netlify.app/How-To-Guides/how-to-create-play

**Review by supminn — CHANGES_REQUESTED** [review]

**Comment by supminn** [comment]
@Nargis21,  Can you add the issue no in the PR description?

File: docs/How-To-Guides/how-to-create-play.md (hunk 1 above)
Comment (Nargis21) [reply docs/How-To-Guides/how-to-create-play.md]: @supminn  Done, check it once now.

**Review by supminn — APPROVED** [review]
Looks good!

File: docs/contribution-guide.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ We also love `typo` issues. If you find a typo somewhere, get in touch us and cr
 - Found a bug. Great! Go ahead and open a [bug report](https://github.com/reactplay/react-play/issues/new?assignees=&labels=bug&template=bug-report.yml&title=%F0%9F%90%9B+%5BBug+report%5D%3A+)
 - You could also improve the [docs](https://github.com/reactplay/docs) which would help us immensely.
 - You could give us feedback, suggestions and help us improve our UX.
-- Have interest in interacting with public, you are welcome to manage our social media handles like [Twitter](https://twitter.com/ReactPlayIO) and [LinkedIn](https://www.linkedin.com/company/reactplay/).
+- Have interest in interacting with public, you are welcome to manage our social media handles like [Twitter](https://twitter.com/ReactPlayIO), [LinkedIn](https://www.linkedin.com/company/reactplay/) and [Facebook](https://web.facebook.com/groups/reactplay).
```
Comment (atapas) [inline docs/contribution-guide.md:32]: Make it like this:

Have an interest in interacting with the public? You are welcome to manage our social media handles like [Twitter](https://twitter.com/ReactPlayIO), [LinkedIn](https://www.linkedin.com/company/reactplay/), and [Facebook](https://web.facebook.com/groups/reactplay).

```
Have an interest in interacting with the public? You are welcome to manage our social media handles like [Twitter](https://twitter.com/ReactPlayIO), [LinkedIn](https://www.linkedin.com/company/reactplay/), and [Facebook](https://web.facebook.com/groups/reactplay).
```

**Review by atapas — CHANGES_REQUESTED** [review]
@Nargis21 A small change

File: docs/contribution-guide.md (hunk 2 above)
Comment (supminn) [reply docs/contribution-guide.md:32]: Hi @Nargis21 
Could you replace the sentence in the way Tapas has mentioned above? Although the content looks the same, it has a few changes w.r.t articles and punctuation.
thanks

Comment (Nargis21) [reply docs/contribution-guide.md:32]: Sure I will do

Comment (Nargis21) [reply docs/contribution-guide.md]: @supminn I pushed the updated code. Kindly check it once and let me know if need anything else.

**Review by supminn — APPROVED** [review]
LGTM

**Review by atapas — APPROVED** [review]

**Pull request merged**